# Chinese (Simplified, China) translations for PROJECT.
# Copyright (C) 2025 ORGANIZATION
# This file is distributed under the same license as the PROJECT project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2025.
#
msgid ""
msgstr ""
"Project-Id-Version: PROJECT VERSION\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2025-08-21 22:15+0800\n"
"PO-Revision-Date: 2025-08-21 13:12+0800\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: zh_Hans_CN\n"
"Language-Team: zh_Hans_CN <LL@li.org>\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.17.0\n"

#: routers/v1/orsdm.py:165
msgid ""
"Task already exists, not allowed to duplicate one, please check odm "
"server."
msgstr "任务已存在，不允许重复创建，请检查ODM服务器。"

#: routers/v1/orsdm.py:179
msgid ""
"No images found in the specified folder. and the job has been removed "
"from the odm."
msgstr "在指定文件夹中未找到图像文件，该任务已从ODM服务器移除。"

#: routers/v1/orsdm.py:247
msgid "Odm task not found"
msgstr "未找到ODM任务"

#: routers/v1/orsdm.py:292 routers/v1/orsdm.py:340 routers/v1/orsdm.py:394
#: routers/v1/orsdm.py:452
msgid "Job not found"
msgstr "未找到任务"

#: routers/v1/orsdm.py:301
msgid "Failed to cancel celery task."
msgstr "取消Celery任务失败"

#: routers/v1/orsdm.py:304
msgid "Failed to remove ODM task from odm server."
msgstr "从ODM服务器移除任务失败"

#: routers/v1/orsdm.py:344
msgid "ODM task must be completed, failed, or canceled before it can be removed."
msgstr "ODM任务必须处于完成、失败或已取消状态才能被移除"

#: routers/v1/orsdm.py:379
msgid "Orthophoto tif file not provided"
msgstr "未提供正射影像tif文件"

#: routers/v1/orsdm.py:386
msgid "Orthophoto tif file not found"
msgstr "未找到正射影像tif文件"

#: routers/v1/orsdm.py:535 routers/v1/orsdm.py:646 routers/v1/orsdm.py:726
msgid "Report not found, please try again later."
msgstr "报告尚未生成，请稍后再试"

#: routers/v1/orsdm.py:649
msgid "Report is already uploaded."
msgstr "报告已上传"

#: routers/v1/orsdm.py:736
msgid "Report is not running. Cannot cancel."
msgstr "报告未在运行中，无法取消"

#: utils/cersdm.py:482
#, python-format
msgid ""
"Committed Cloud ODM report failed. Error code: %(error_code)d, Message: "
"%(error_message)s"
msgstr "提交云端ODM报告失败。错误代码：%(error_code)d，错误信息：%(error_message)s"

#: utils/cersdm.py:484
msgid "Unknown error"
msgstr "未知错误"
